src/executor.rs

```diff
@@ -635,9 +635,9 @@ fn parse_result(
         | CoreTypeConcrete::StarkNet(_)
         | CoreTypeConcrete::Uint128MulGuarantee(_)
         | CoreTypeConcrete::Circuit(_)
-        | CoreTypeConcrete::RangeCheck96(_) => todo!(),
+        | CoreTypeConcrete::RangeCheck96(_) => native_panic!("not yet implemented as results"),
```

Comment:
The `RangeCheck96` is a builtin, therefore should go in the builtins category (the block just above, after `RangeCheck`).